src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -461,3 +463,11 @@ def load_command_table(self, _):
     with self.command_group('staticwebapp secrets', custom_command_type=staticsite_sdk) as g:
         g.custom_command('list', 'list_staticsite_secrets')
         g.custom_command('reset-api-key', 'reset_staticsite_api_key', supports_no_wait=True)
+
+    with self.command_group('logicapp') as g:
+        g.custom_command('delete', 'delete_function_app')
+
+    with self.command_group('logicapp', custom_command_type=logicapp_custom) as g:
+        g.custom_command('create', 'create_logicapp', exception_handler=ex_handler_factory())
+        g.custom_command('list', 'list_logicapp', table_transformer=transform_web_list_output)
```

Comment:
Are you able to get `list` to work, when you have no logicapps in your sub? I seem to be getting error: 
![image](https://user-images.githubusercontent.com/14339314/132598089-9086b80f-9c5d-4a97-8749-ef85bf1886d4.png)

Reply:
It repro's on cloudshell for me for `functionapp list` as well

Reply:
Hmm that's strange, works for me:

`
(env) PS C:\Users\paaggarwal\Desktop\azure-cli> az logicapp list

[]
`